ansible/roles/optimizations/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Disable daily upgrades
+  systemd:
+    name: "{{ item }}"
+    state: disabled
```

Comment:
You have made a mistake here, according to [documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_module.html), `state` parameter can't take `disable`. For disable a service you must replace it by `stopped`  and add this line : `enabled: false
### Final code should be : 
```yaml
---

- name: Disable daily upgrades
  systemd:
    name: "{{ item }}"
    state: stopped
    enabled: false
    daemon-reload: yes
  with_items:
    - 'apt-daily-upgrade.timer'
    - 'apt-daily.timer'
    - 'apt-daily.service'
 ```